string ObjectName
int8 ObjectIndex
---
bool Found
